R/Connection.R

```diff
@@ -109,8 +109,13 @@ setClass(
 #' the columns `data_type`, `column_size`, and `decimal_digits` are used.  An
 #' implementation is not necessary for [dbWriteTable()] to work.
 #' @param conn OdbcConnection
-#' @param name table we wish to get information on
+#' @param name Identifier for table we wish to get information on.  Interpreted
+#' as a SQL search pattern: underscores and percent signs are interpreted as
+#' wild cards.  Related to `exact` parameter.
 #' @param ... additional parameters to methods
+#' @param exact Se to TRUE if any non-null identifier arguments are to be interpreted
```

Comment:
I think my only remaining question is if we want to use `exact = TRUE` (or maybe `escape = TRUE` or `pattern = FALSE`) or `I()`. The advantage of an argument is that it makes it quite clear in the function signature; the advantage of `I()` is that it keeps the two concerns very closely coupled.

Reply:
Good question:

Can go either way on this one, but maybe slightly in favor of the argument because the user may be interacting with the API by passing in a `DBI::Id` or `DBI::SQL` argument.

Haven't thought too hard about how to pass the `I()` through those containers / feels like there is a way but it might start to stretch the proposition.